orttraining/orttraining/test/python/orttraining_transformer_trainer.py

```diff
@@ -20,10 +20,18 @@
 from transformers.modeling_utils import PreTrainedModel
 from transformers.training_args import TrainingArguments
 
+# import sys
+# sys.path.append("/bert_ort/liqun/onnxruntime/build/Linux/Debug")
+
 import onnxruntime
 from orttraining_test_bert_postprocess import postprocess_model
 from onnxruntime.capi.ort_trainer import ORTTrainer, LossScaler, ModelDescription, IODescription
 
+from onnxruntime.experimental import _utils, amp, optim, orttrainer, TrainStepInfo,\
```

Comment:
This can be broken into multiple import statements for formatting and readability.